@xen-orchestra/lite/src/components/AddNetwork.tsx

```diff
@@ -0,0 +1,255 @@
+}
+
+const BOND_MODE = ['active-backup', 'balance-slb', 'lacp']
+
+const BUTTON_STYLES = {
+  marginRight: '1em',
+  width: 'fit-content',
+}
+
+const OPTION_PIF_RENDERER = (pif: Pif, { pifsMetrics }: { pifsMetrics: Computed['pifsMetrics'] }) =>
+  `${pif.device} (${pifsMetrics?.find(metrics => metrics.$ref === pif.metrics)?.device_name ?? 'unknown'})`
+
+const INPUT_STYLES = {
+  marginBottom: '1em',
+  width: '100%',
+}
+
+const getInitialFormState = (): State['form'] => ({
+  bondMode: '',
+  description: '',
+  isBonded: false,
+  isEmptyLabel: false,
+  mtu: '',
+  nameLabel: '',
+  pifsId: '',
+  vlan: '',
+})
+
+const AddNetwork = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {
+    initialState: () => ({
+      form: getInitialFormState(),
+    }),
+    computed: {
+      pifs: state => state.objectsByType.get('PIF'),
+      pifsMetrics: state => state.objectsByType.get('PIF_metrics'),
+      collection: state =>
+        state.pifs
+          ?.filter(pif => pif.VLAN === -1 && pif.bond_slave_of === 'OpaqueRef:NULL' && pif.host === pif.$pool.master)
+          .sortBy(pif => pif.device)
+          .valueSeq()
+          .toArray(),
+    },
+    effects: {
+      createNetwork: async function () {
+        const { bondMode, description, mtu, nameLabel, pifsId, vlan } = this.state.form
+        if (nameLabel.trim() === '') {
+          this.state.form = {
+            ...this.state.form,
+            isEmptyLabel: true,
+          }
+          return
+        }
+        try {
+          await this.state.xapi.createNetworks([
+            {
+              MTU: +mtu,
+              name_description: description,
+              name_label: nameLabel,
+              VLAN: +vlan,
+              bondMode: bondMode === '' ? undefined : bondMode,
+              pifsId: pifsId === '' ? undefined : Array.isArray(pifsId) ? pifsId : [pifsId],
+            },
+          ])
+          this.effects.resetForm()
+        } catch (error) {
+          console.error(error)
+          if (error instanceof Error) {
+            alert({
+              icon: 'exclamation-triangle',
+              message: <p>{error.message}</p>,
+              title: <IntlMessage id='networkCreation' />,
+            })
+          }
+        }
+      },
+      handleChange: function ({ target: { name, value } }) {
+        // Reason why form values are initialized with empty string and not a undefined value
+        // Warning: A component is changing an uncontrolled input to be controlled.
+        // This is likely caused by the value changing from undefined to a defined value,
+        // which should not happen. Decide between using a controlled or uncontrolled input
+        // element for the lifetime of the component.
+        // More info: https://reactjs.org/link/controlled-components
+        const { form } = this.state
+        if (name === 'nameLabel') {
+          form.isEmptyLabel = false
+        }
```

Comment:
Why this change?

Reply:
It's related to [this](https://github.com/vatesfr/xen-orchestra/pull/5863/files#diff-d8b019cde681b4e8767af938e54bab866da811d38ef0a4eb0e3826785012a57dR195).